### PR TITLE
Split test acceptanceTests

### PR DESCRIPTION
**Pull request by matkt**
Signed-off-by: Karim TAAM <karim.t2am@gmail.com>

## PR description

Even if we set up tests in parallel we just create several docker that are on the same machine and that therefore queries the same source of entropy. I tried to split the test into different steps.

I  tried 5 times and it's fine everytime

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -185,6 +185,53 @@ jobs:
...
+              | sed 's@/@.@g' \
+              | sed 's/.\{5\}$//' \
+              | circleci tests split --split-by=timings --timings-type=classname)
+            # Format the arguments to "./gradlew test"
+            GRADLE_ARGS=$(echo $CLASSNAMES | awk '{for (i=1; i<=NF; i++) print "--tests",$i}')
+            ./gradlew --no-daemon acceptanceTest $GRADLE_ARGS
+      - capture_test_results
+      - capture_test_logs
+
+  acceptanceTestsPrivacy:
```
Comment (macfarla) [inline .circleci/config.yml:199]: make this a required step

File: .circleci/config.yml [hunk 2]
```diff
@@ -185,6 +185,53 @@ jobs:
...
+              | sed 's@/@.@g' \
+              | sed 's/.\{5\}$//' \
+              | circleci tests split --split-by=timings --timings-type=classname)
+            # Format the arguments to "./gradlew test"
+            GRADLE_ARGS=$(echo $CLASSNAMES | awk '{for (i=1; i<=NF; i++) print "--tests",$i}')
+            ./gradlew --no-daemon acceptanceTest $GRADLE_ARGS
+      - capture_test_results
+      - capture_test_logs
+
+  acceptanceTestsPermissioning:
```
Comment (macfarla) [inline .circleci/config.yml:222]: make this a required step

File: .circleci/config.yml [hunk 3]
```diff
@@ -185,6 +185,53 @@ jobs:
...
+    parallelism: 8
```
Comment (macfarla) [inline .circleci/config.yml:223]: looking at the output, this step now only takes 3 min so could prob reduce the concurrency but we can tune that later (not sure how this affects the credit usage)

**Review by ajsutton — COMMENTED** [review]
This doesn't actually change where the containers come from - whether you use parallel or separate steps CircleCI will just allocate an executor from its pool. 

All this does is make the config more complex and makes the balancing of tests across containers less effective.

**Pull request closed without merging**